test/Apache-NMS-AMQP-Interop-Test/Apache-NMS-AMQP-Interop-Test.csproj

```diff
@@ -16,11 +16,11 @@ under the License.
 -->
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net462;netcoreapp2.2</TargetFrameworks>
+    <TargetFrameworks>net462;netcoreapp3.1</TargetFrameworks>
```

Comment:
can you make it so 3.1 is in addtion to existing, not replacing.

Reply:
but netcoreapp2.2 is out of support